arangod/Aql/Executor/SortedCollectExecutor.cpp

```diff
@@ -89,9 +89,7 @@ void SortedCollectExecutor::CollectGroup::reset(InputAqlItemRow const& input) {
 
   if (!groupValues.empty()) {
     for (auto& it : groupValues) {
-      auto mem = it.memoryUsage();
       it.destroy();
```

Comment:
Different PR:
But shouldn't we check if this needs to be destroyed in the first place?